issues_arm64_test.go

```diff
@@ -0,0 +1,34 @@
+//go:build darwin && arm64
+// +build darwin,arm64
```

Comment:
`+build darwin,arm64` is equivalent to `go:build darwin || arm64`. You need 2 lines for `&&`:
```
//go:build darwin && arm64
// +build darwin
// +build arm64
```

The `arm64` build tag is redundant with the file naming (`_arm64_test.go`), I would remove it.